tables/isolates.csv

```diff
@@ -1332,3 +1332,12 @@ B.1.620 Spike,B.1.620,FALSE,NULL,NULL,NULL,TRUE
 B.1.620 Spike:+845I,B.1.620,FALSE,NULL,NULL,NULL,TRUE
 S:19R+452R+478K+614G+681R+950N,NULL,FALSE,NULL,NULL,NULL,TRUE
 S:19del,NULL,TRUE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:E147K,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:R152W,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:L157F,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:V210I,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:S257G,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:H339D,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:S446G,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:K460N,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE
+BA.2.75 Spike:Q493R,Omicron/BA.2.75,FALSE,NULL,NULL,NULL,TRUE

```

Comment:
@zachandfox Please add `var_name`, Omicron/BA.2.75/H339D, Omicron/BA.2.75/S446G, Omicron/BA.2.75/K460N, Omicron/BA.2.75/Q493R,  in the `variants` table. Then use them here to link to the corresponding `iso_name`.